store/copr/coprocessor.go

```diff
@@ -1479,6 +1511,19 @@ func (worker *copIteratorWorker) handleBatchCopResponse(bo *Backoffer, rpcCtx *t
 		}
 		appendRemainTasks(t.task)
 	}
+	if regionErr := resp.GetRegionError(); regionErr != nil && regionErr.ServerIsBusy != nil &&
```

Comment:
We might assert `!(busyThresholdFallback && len(bachResps) > 0)`, protecting it from future change.

Reply:
I returned an error when this happened.